config/crd/patches/cainjection_in_keymanagementproviders.yaml

```diff
@@ -0,0 +1,7 @@
+# The following patch adds a directive for certmanager to inject CA into the CRD
```

Comment:
all previous CRDS have equivalent files in the patches/ directory; however, it is not being used in our actual chart since these cainjection* files are used if you enable cert manager. Currently, we do not enabled cert manager.